From: Benjamin Otte <otte@redhat.com>
Date: Fri, 21 Nov 2025 10:08:12 +0100
Subject: Fix a few profiler messges for 32bit

Related: #7883
Part-of: <https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/9170>
(cherry picked from commit 8af38ff007eff69338ff7be2bd7cb22c846d121f)
---
 gdk/gdkmemoryformat.c  | 2 +-
 gdk/gdkvulkancontext.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gdk/gdkmemoryformat.c b/gdk/gdkmemoryformat.c
index 03f3044..04f6057 100644
--- a/gdk/gdkmemoryformat.c
+++ b/gdk/gdkmemoryformat.c
@@ -6620,7 +6620,7 @@ gdk_memory_mipmap_same_format_linear (gpointer data)
     }
 
   ADD_MARK (before,
-            "Mipmap linear (thread)", "size %lux%lu, lod %u, %lu rows",
+            "Mipmap linear (thread)", "size %zux%zu, lod %u, %zu rows",
             mipmap->src_layout.width, mipmap->src_layout.height, mipmap->lod_level, rows);
 }
 
diff --git a/gdk/gdkvulkancontext.c b/gdk/gdkvulkancontext.c
index 2639389..c39774b 100644
--- a/gdk/gdkvulkancontext.c
+++ b/gdk/gdkvulkancontext.c
@@ -1541,7 +1541,7 @@ gdk_vulkan_save_pipeline_cache (GdkDisplay *display)
     }
 
   gdk_profiler_end_markf (begin_time,
-                          "Save Vulkan pipeline cache", "%s size %lu",
+                          "Save Vulkan pipeline cache", "%s size %zu",
                           g_file_peek_path (file), size);
 
   g_object_unref (file);
